docs/documentation/gui.md

```diff
@@ -50,17 +50,17 @@ To install the Dimlpfidex GUI, follow these steps:
             node app.js
```

Comment:
Do we need instruction for `node.js`? This is still happening locally, so I wouldn't miscall it "Web". I'd remove it entirely.

Note: What baout macOS? It is one of our target. De we have a GUI compiled for it?